datafusion/physical-expr/src/aggregate.rs

```diff
@@ -215,6 +224,7 @@ pub struct AggregateFunctionExpr {
     /// Output / return type of this aggregate
     data_type: DataType,
     name: String,
+    sql_name: String,
```

Comment:
I am worried about adding this new String for every single AggregateFunctionExpr even when we are not printing an explain plan -- we are trying try to keep DataFusion's planning time from getting out of hand so if we can avoid forcing a new String  that would be better